wandb/apis/public.py

```diff
@@ -2693,25 +2693,27 @@ def _load(self):
         """
             % ARTIFACT_FRAGMENT
         )
-        response = self.client.execute(
-            query,
-            variable_values={
-                "entityName": self.entity,
-                "projectName": self.project,
-                "name": self.artifact_name,
-            },
-        )
-        if (
-            response is None
-            or response.get("project") is None
-            or response["project"].get("artifact") is None
-        ):
+        try:
```

Comment:
Looks like you should have:
```
response = None
```
before the try since you are checking response outside the try.